packages/transactions/src/OutboundShipment/DetailView/modals/ItemDetails.tsx

```diff
@@ -107,7 +100,10 @@ export const ItemDetails: React.FC<ItemDetailsProps> = ({
           />
         </ModalRow>
         <ModalInputRow
-          inputProps={register('quantity')}
+          inputProps={register('quantity', {
+            required: true,
+            min: { value: 1, message: t('error.greater-than-zero-required') },
```

Comment:
this is valid for `a` etc. Making the input a number type I suppose? Or regex or something for validation?

Reply:
sure. have added a pattern attribute to validate numeric input